sympy/sets/tests/test_fancysets.py

```diff
@@ -379,6 +379,10 @@ def test_ComplexRegion_union():
     assert c1.union(Interval(2, 4)) == Union(c1, Interval(2, 4))
     assert c5.union(Interval(2, 4)) == Union(c5, Interval(2, 4))
 
+    # Issue #10419
+    c1 = ComplexRegion(Interval(0, 1)*Interval(0, 2*S.Pi), polar=True)
+    assert c1.union(Interval(2, 4)) == Union(c1, Interval(2, 4))
```

Comment:
This is not the right test to be added. The thing is the answer should be like this
```
>>> c1.union(Interval(2, 4))
[2, 4] U ComplexRegion( .... )   #  ComplexRegion is c1 here
```
Also there is no point in checking equality of `c1.union(Interval...)` with `Union(c1, Interval...)` since `c1.union(Interval..)` works by calling `Union(c1, Interval..)` itself.

So i think it should be `Union(c1, Interval(2, 4), evaluate=False)` with `evaluate=False` (yes, i think this should be the one)